README.md

```
@@ -1,384 +1,377 @@
+2. Generate API key
+3. Add to `.env`:
+
+```env
+VITE_GEMINI_API_KEY=your_gemini_api_key
 ```
 
----
+### 🌐 Backend Configuration
 
-## 🔑 API Setup
+**Option A: Hosted Backend (Recommended)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)